remote/src/main/java/com/example/appengine/remote/RemoteApiSeedPageViewData.java

```diff
@@ -0,0 +1,215 @@
+package com.example.appengine.remote;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.GeoPt;
+import com.google.appengine.tools.remoteapi.RemoteApiInstaller;
+import com.google.appengine.tools.remoteapi.RemoteApiOptions;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+public class RemoteApiSeedPageViewData {
+  public static void main(String[] args) throws IOException {
+    String serverString = args[0];
+    System.out.println(serverString);
+    RemoteApiOptions options;
+    if (serverString.equals("localhost")) {
+      options = new RemoteApiOptions().server(serverString, 8080).useDevelopmentServerCredential();
+    } else {
+      options = new RemoteApiOptions().server(serverString, 443).useApplicationDefaultCredential();
+    }
+    RemoteApiInstaller installer = new RemoteApiInstaller();
+    installer.install(options);
+    try {
+      DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+      Entity pageViews1 = new Entity("PageViews");
+      pageViews1.setProperty("restaurantKey", 1);
+      pageViews1.setProperty("restaurantName", "Wildfire Restaurant");
+      pageViews1.setProperty("year", 2020);
+      pageViews1.setProperty("week", 26);
+      pageViews1.setProperty("count", 39);
+
+      Entity pageViews2 = new Entity("PageViews");
+      pageViews2.setProperty("restaurantKey", 1);
```

Comment:
Do these values correspond with the actual keys that are seeded for the restaurants themselves?